src/benchmarks/micro/corefx/System.Collections/Sort.cs

```diff
@@ -5,31 +5,56 @@
 using System.Collections.Generic;
 using System.Linq;
 using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Diagnostics.Windows.Configs;
 using BenchmarkDotNet.Extensions;
 using MicroBenchmarks;
 
 namespace System.Collections
 {
+    // TODO: Where should the below types reside?
+
+    public readonly struct IntStruct : IComparable<IntStruct>
+    {
+        readonly int _value;
+
+        public IntStruct(int value) => _value = value;
+
+        public int CompareTo(IntStruct other) => _value.CompareTo(other._value);
+    }
+
+    public class IntClass : IComparable<IntClass>
```

Comment:
@billwert where should these extra types reside?

Reply:
I think if they're just used by this benchmark in the file is fine. Putting them in a file next to Sort.cs is also fine.